scripts/gen-deps-list.js

```diff
@@ -38,33 +38,35 @@ async function main() {
     throw new Error(`Could not find changelog block in ${changelogConfig.path}`)
   }
 
-  const lines = block.split(/\s*\r?\n\s*/).map(line => {
-    const items = trim(line, '- ').split(/\s+/)
+  block.split(/\r?\n/).forEach(rawLine => {
+    const line = rawLine.trim()
 
-    if (items.length !== 2) {
-      throw new Error(`Invalid line: "${line}"`)
+    if (!line) {
```

Comment:
It does not bother me much because it's a standalone script, but IMHO it's a good practice to explicit comparisons and not use implicit boolean casts, because it helps avoiding bugs and document the variables (tremendously useful when refactoring old code).

Reply:
But as I said, it does not bother me in this file, you don't have to change your code :slightly_smiling_face: